src/main/java/ch/njol/skript/sections/EffSecStructurePlace.java

```diff
@@ -0,0 +1,191 @@
+import org.jetbrains.annotations.Nullable;
+import org.jetbrains.annotations.NotNull;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.config.SectionNode;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.EffectSection;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.lang.Trigger;
+import ch.njol.skript.lang.TriggerItem;
+import ch.njol.skript.variables.Variables;
+import ch.njol.util.Kleenean;
+
+@Name("Structure Place")
+@Description({
+	"Places a structure. This can be used as an effect and as a section.",
+	"If it is used as a section, the section is run before the structure is placed in the world.",
+	"You can modify the place settings like if it should include entities and any rotation/mirror option you want.",
+	"For more info on the section settings see <a href='expressions.html#ExprStructureSectionInfo'>Structure Place Settings</a>"
+})
+@Examples({
+	"place structure \"minecraft:end_city\" at player's location without entities:",
+		"\tset integrity to 0.9",
+		"\tset pallet to -1 # -1 for random pallet",
+		"\tset pallet to -1 # random pallet",
+		"\tset rotation to counter clockwise 90",
+		"\tset mirror to none # already the default not required"
+})
+@Since("INSERT VERSION")
+public class EffSecStructurePlace extends EffectSection {
+
+	public class StructurePlaceEvent extends Event {
+
+		private StructureRotation rotation = StructureRotation.NONE;
+		private Mirror mirror = Mirror.NONE;
+		private final Structure structure;
+		private float integrity = 1F;
+		private boolean entities;
+		private int pallet = 0;
+
+		public StructurePlaceEvent(Structure structure, boolean entities) {
+			this.structure = structure;
+			this.entities = entities;
+		}
+
+		public Structure getStructure() {
+			return structure;
+		}
+
+		public boolean includeEntities() {
+			return entities;
+		}
+
+		public void setIncludesEntities(boolean entities) {
+			this.entities = entities;
+		}
+
+		public StructureRotation getRotation() {
+			return rotation;
+		}
+
+		public void setRotation(StructureRotation rotation) {
+			this.rotation = rotation;
+		}
+
+		public Mirror getMirror() {
+			return mirror;
+		}
+
+		public void setMirror(Mirror mirror) {
+			this.mirror = mirror;
+		}
+
+		public int getPallet() {
+			return pallet;
+		}
+
+		public void setPallet(int pallet) {
+			this.pallet = pallet;
+		}
+
+		public float getIntegrity() {
+			return integrity;
+		}
+
+		public void setIntegrity(float integrity) {
+			this.integrity = integrity;
+		}
+
+		@Override
+		@NotNull
+		public HandlerList getHandlers() {
+			throw new IllegalStateException();
+		}
+	}
+
+	static {
+		if (Skript.classExists("org.bukkit.structure.Structure"))
+			Skript.registerSection(EffSecStructurePlace.class, "place %structure% at %locations% [without :entities]");
```

Comment:
I don't think this parse tag will work properly. It will have a parse tag `without ` for the syntax `entities`